proof/refine/ARM/Finalise_R.thy

```diff
@@ -4289,6 +4358,40 @@ lemma sched_context_maybe_unbind_ntfn_corres:
      apply (wpsimp wp: get_simple_ko_wp getNotification_wp split: option.splits)+
   done
 
+lemma replyRemove_corres:
+  "corres dc (invs and tcb_at tptr) (invs' and K (rptr' = rptr))
+      (reply_remove tptr rptr) (replyRemove rptr' tptr)"
+  apply (clarsimp simp: reply_remove_def replyRemove_def)
+  sorry
+
+lemma replyClear_corres:
+  "corres dc
+          (invs and valid_ready_qs and st_tcb_at is_reply_state tp)
+          (invs' and st_tcb_at' (\<lambda>st. replyObject st = Some rptr) tp)
+          (do
```

Comment:
Nitpick: perhaps change the indentation of this `do` block?

Reply:
What indentation would you prefer?

Reply:
Do we want the first line of the `do` block on the same line as the first `do`? Not a big deal, of course